start-from-zero/plus.js

```diff
@@ -0,0 +1,11 @@
+
+var plus = document.querySelector('.plus-sign');
```

Comment:
Could you run me through what this is doing? Also, are you certain that when this script runs, the proper DOM elements will already be in place?